### PR TITLE
Adding mattermost as datasource enabling Grafana

### PR DESCRIPTION
**Pull request by tapaswenipathak**
Hi folks,

I didn't find the [GSoD'19 tasks](https://forum.mattermost.org/t/google-season-of-docs-2019/7136) open in any repository, the tasks described on the forum are only internal organisation work. I will be closing few tickets from docs repository or any useful addition I find. Please let me know any open task that I can do in GSoD'19 application period. Thanks.

ref https://github.com/mattermost/docs/issues/2297.

### CODE REVIEW

**Comment by amyblais** [comment]
Hi @tapaswenipathak, thank you for this PR!

It looks like the image doesn't render in the preview - please upload any new images through docs > source > images (https://github.com/mattermost/docs/tree/master/source/images). Let me know any questions.

File: source/deployment/metrics.rst [hunk 1]
```diff
@@ -86,6 +86,10 @@ For install instructions, see `Grafana install guides <http://docs.grafana.org/i
 
 3 - The default HTTP port is ``3000`` and default username and password are ``admin``.
 
+4 - Adding a Mattermost data source
```
Comment (amyblais) [inline source/deployment/metrics.rst]:
```suggestion
4 - Add a Mattermost data source:
```

**Comment by wiersgallak** [comment]
Hi @tapaswenipathak  Thanks for checking out the process and providing feedback. I am in the process of creating more issues for the GSoD to better guide you on the projects and what we would like to see.  Feel free to reach out if you have any other questions!

**Comment by tapaswenipathak** [comment]
On Thu, May 16, 2019 at 2:17 AM Katie Wiersgalla
<notifications@github.com> wrote:
>
> Hi @tapaswenipathak Thanks for checking out the process and providing feedback. I am in the process of creating more issues for the GSoD to better guide you on the projects and what we would like to see. Feel free to reach out if you have any other questions!

Hello, this would be useful, thanks. I think I had a ping from the
channel, I will speak w/ you there

>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.



--
Tapasweni

File: source/deployment/metrics.rst [hunk 2]
```diff
@@ -86,6 +86,10 @@ For install instructions, see `Grafana install guides <http://docs.grafana.org/i
 
 3 - The default HTTP port is ``3000`` and default username and password are ``admin``.
 
+4 - Add a Mattermost data source:
```
Comment (jasonblais) [inline source/deployment/metrics.rst]:
```suggestion
4 - A Mattermost data source should be added, with settings defined in the screenshot below.
```

**Pull request merged**